core/src/main/java/hudson/PluginManager.java

```diff
@@ -819,100 +819,102 @@ public void dynamicLoad(File arc) throws IOException, InterruptedException, Rest
      */
     @Restricted(NoExternalUse.class)
     public void dynamicLoad(File arc, boolean removeExisting) throws IOException, InterruptedException, RestartRequiredException {
-        LOGGER.info("Attempting to dynamic load "+arc);
```

Comment:
Now you are going to have use [this](https://github.com/jenkinsci/jenkins/pull/2732/files?w=1) to understand what is going on.